ow_lander/urdf/lander.xacro

```diff
@@ -544,12 +544,12 @@
     </visual>
     <selfCollide>1</selfCollide>
     <collision>
+      <max_contacts>1</max_contacts>

```

Comment:
I'm concerned about this because it will might result in unrealistic collisions. How is the one contact point chosen--Is it just the contact point where penetration is deepest? And what if a very different contact point is chosen on the next frame--couldn't that cause jerking or popping motions?

Reply:
I was hesitant as well and I wanted to hear your opinion. From testing, the scoop-terrain interaction doesn't seem to change after reducing the max_contacts: I didn't observe any jerking/popping. I am not sure how the contact point is chosen. When setting max_contacts to a number greater than 1, the ODE error shows up. If we decide to run the sim at timestep=0.005 s I might be able to increase the contact points without getting the error.

Reply:
I would leave this for a separate ticket as it should be tested in more situations before settling on a solution. The LCP errors are annoying but don't seem to cause any real damage. Please make a ticket for suppressing LCP errors and include two possible solutions: 1) max_contacts, or 2) https://github.com/osrf/gazebo/blob/6fd426b3949c4ca73fa126cde68f5cc4a59522eb/gazebo/physics/ode/ODEPhysics_TEST.cc#L336
Or there may be other solutions we have not discovered.

Reply:
Done!